website/docs/getting-started/for-designers/partials/best-practices.md

```diff
@@ -1,87 +1,101 @@
+Effectively use Helios components and foundations by following these best practices.
 
-## Component availability
+For documentation around broader Figma best practices within HashiCorp design teams, reach out to the HashiCorp [Design Ops team](https://sites.google.com/hashicorp.com/designknowledgehub/design-system) or visit the [#design-ops slack channel](https://hashicorp.slack.com/archives/C029GL8GJDV).
 
-**Structure** is the legacy set of components and foundations used to connect HashiCorp products and consists Ember components and foundational styles paired with a limited set of Figma counterparts. **Structure is no longer supported and is in the process of deprecation and sunsetting.**
+## When to use Helios
 
-As we work to achieve parity between Structure and Helios, use this decision tree to understand how to move forward within your project:
+!!! Info
 
-![Decision tree flow chart](/assets/getting-started/designers/hds-decision-tree.png)
+**What is structure?**
 
-**Does the element (or a comparable alternative) exist in Helios?**
+Structure is the legacy design system used primarily by Cloud UI. It is no longer supported and being sunset.
+!!!
 
-If so, great! Using this element ensures it will be supported in future versions of Helios and that it shares a common visual language with the rest of the elements.
+Use this decision tree to understand when to use a component from Helios, from Structure, or when to create your own local component or pattern.
 
-**If no, does the element (or a comparable alternative) exist in Structure?**
+![Decision tree flow chart](/assets/getting-started/designers/hds-decision-tree.png)
 
-If yes, use this component with the understanding that:
+### Element exists in Helios
 
-1. Structure will be deprecated/sunsetted in the future
-2. Not all Figma components are built in code
-3. There is no longer a support structure in place if errors/bugs arise
+If the component or pattern exists in Helios, use it. This ensures it will be supported in future versions of Helios and that it shares a common visual language with the rest of the elements.
 
-If no, or having future support is a must: design and build your own local element using Helios foundations (color, typography, elevation, etc) as a starting point.
+### Element exists in Structure
 
-### Local components
+If the component or pattern doesn't exist in Helios, but does exist in Structure, use the Structure version, but do so sparingly and with caution because:
```

Comment:
```suggestion
If the component or pattern doesn’t exist in Helios, but does exist in Structure, use the Structure version, but do so sparingly and with caution because:
```